contracts/utilities/TokenFeeCollector.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../interfaces/IERC20.sol";
+import "../lib/Ownable.sol";
+import "../lib/SafeMath.sol";
+
+contract TokenFeeCollector is Ownable {
+    using SafeMath for uint;
+
+    // this means 100% so 1000000 is 1%, 100000 is 0.1%, etc
+    uint private constant maxBips = 100000000;
+
+    struct TokenInfo {
+        bool isValue;
```

Comment:
could we use this flag to pause or disable payments (e.g. in case of configuring new payees or distributions)

Reply:
it's possible but it would have a nasty side-effect, the admin would be able to call registerToken again and completely replace all existing data

Reply:
I think for configuring new payees, it might be a good idea to call the sweep first, then add again, little damage could be done if the majority of the balance was already distribute

Reply:
it's also safe to assume payees will collaborate with the admin as the admin has the power to remove him if he misbehaves